sqlalchemy_ibmi/base.py

```diff
@@ -749,49 +749,19 @@ def create_connect_args(self, url):
         opts = url.translate_connect_args(username='user')
         opts.update(url.query)
         keys = opts
```

Comment:
Also, I think we should add some explanation to the `translate_connect_args` call. According to the docs, that will convert the attributes of the Url to a dictionary, eg. url.database -> `{ 'database': '...'}` while allowing the caller to specify an alternate key name for attributes - in this case we map url.username to the user key, eg. `{'user': '...'}`.

I think it would be helpful to do all of the attributes to the IBM i Access keywords. Looks like host is the only other one:
```python
# We map the SQLAlchemy arguments to the corresponding ODBC driver keywords
keys = url.translate_connect_args(username='userid', host='system')
```

Reply:
It looks like [pyodbc.connect](https://github.com/mkleehammer/pyodbc/wiki/Module#connect) will already adjust those for us, so I guess we don't need to pass anything to `translate_connect_args`.

Reply:
@naveen-ram I think it actually makes more sense to have it be called opts over keys. My initial suggestion of calling it keys was based on the rest of the code using keys, but most of that code was removed, so there's really not many places to update any more.